planning/motion_velocity_smoother/src/smoother/analytical_jerk_constrained_smoother/velocity_planning_utils.cpp

```diff
@@ -199,13 +199,14 @@ bool calcStopVelocityWithConstantJerkAccLimit(
   }
 
   // for debug
-  std::stringstream ss;
+  RCLCPP_DEBUG(rclcpp::get_logger("velocity_planning_utils"), "Calculate stop velocity.");
   for (unsigned int i = 0; i < ts.size(); ++i) {
-    ss << "t: " << ts.at(i) << ", x: " << xs.at(i) << ", v: " << vs.at(i) << ", a: " << as.at(i)
-       << ", j: " << js.at(i) << std::endl;
+    RCLCPP_DEBUG_STREAM(
```

Comment:
I think the source code for `RCLCPP_DEBUG_STREAM` is generated from https://github.com/ros2/rclcpp/blob/22df1d593a3e77917db8a7d7b20f57aee80d7e55/rclcpp/resource/logging.hpp.em#L143 where it uses `std::stringstream` so I am not sure this PR will improve performance.

Reply:
@maxime-clem Thank you for your information. I will change the code to use `RCLCPP_DEBUG` with format string.

Reply:
@maxime-clem I fixed.
https://github.com/autowarefoundation/autoware.universe/pull/7131/commits/8a87a6ae145cc390d3889be6594f3e50af57c9da